src/Impl/RestStreamxClient.php

```diff
@@ -18,13 +18,12 @@ class RestStreamxClient implements StreamxClient
      */
     public function __construct(
         string $serverUrl,
-        string $ingestionEndpointPath,
+        string $ingestionEndpointBasePath,
         ?string $authToken,
         HttpRequester $httpRequester,
         JsonProvider $jsonProvider
     ) {
-        $ingestionEndpointUri = HttpUtils::buildAbsoluteUri($serverUrl . $ingestionEndpointPath);
```

Comment:
Note: any exception if the url is invalid, will now be thrown not at client creation, but at publisher creation